templates/workload.template.yaml

```diff
@@ -229,11 +229,11 @@ Outputs:
 Parameters:
   MasterUsername:
     Type: String
-    Description: Master user name for the Amazon Redshift cluster.
+    Description: Master user name for the Amazon Redshift cluster. The user name must be lowercase, begin with a letter, contain only alphanumeric characters, '_', '+', '.', '@', or '-', and be less than 128 characters.
```

Comment:
Instead of just adding a description, can you also add constraint so that it can be enforced?

Reply:
Done, see [commit](https://github.com/aws-quickstart/quickstart-aws-utility-meter-data-analytics-platform/commit/8a814609db2ebe3e373be1b87dd65ca543fbccf5)